Gemfile

```diff
@@ -3,11 +3,3 @@ source 'https://rubygems.org'
 gemspec
 
 gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
-
-group :development do
-  gem 'rspec'
```

Comment:
I use rubocop locally, like you use pry. Cucumber uses the expect method of
rspec. Tests will not run in travis-ci if rspec is not included.

Thanks,

Dave McNulla


On Fri, Apr 8, 2016 at 10:42 PM, Павел Скрылёв (Malo Skrylevo) <
notifications@github.com> wrote:

> In Gemfile <https://github.com/dmcnulla/ntp/pull/1#discussion_r59110829>:
>
> > @@ -3,11 +3,3 @@ source 'https://rubygems.org'
> >  gemspec
> >
> >  gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
> > -
> > -group :development do
> > -  gem 'rspec'
>
> @dmcnulla <https://github.com/dmcnulla> Have you use the rspec and
> rubocop just locally for the project?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/dmcnulla/ntp/pull/1/files/24a259dc48a3962964a53e46b3778e6441c8ae2c..1ec80f8d6a1bf2887d99a8bd6f079fbd1167a85d#r59110829>
>